src/LBFactory.sol

```diff
@@ -70,89 +86,132 @@ contract LBFactory is PendingOwnable, ILBFactory {
         emit FlashLoanFeeSet(0, flashLoanFee);
     }
 
-    // TODO: Natspecs
-
+    /**
+     * @notice Get the minimum bin step a pair can have
+     * @return minBinStep
+     */
     function getMinBinStep() external pure returns (uint256 minBinStep) {
         return _MIN_BIN_STEP;
     }
 
+    /**
+     * @notice Get the maximum bin step a pair can have
+     * @return maxBinStep
+     */
     function getMaxBinStep() external pure returns (uint256 maxBinStep) {
         return _MAX_BIN_STEP;
     }
 
+    /**
+     * @notice Get the protocol fee recipient
+     * @return feeRecipient
+     */
     function getFeeRecipient() external view returns (address feeRecipient) {
         return _feeRecipient;
     }
 
+    /**
+     * @notice Get the maximum fee percentage for flashLoans
+     * @return maxFee
+     */
     function getMaxFee() external pure returns (uint256 maxFee) {
         return _MAX_FEE;
     }
 
+    /**
+     * @notice Get the maximum protocol share percentage
+     * @return maxProtocolShare
+     */
     function getMaxProtocolShare() external pure returns (uint256 maxProtocolShare) {
         return _MAX_PROTOCOL_SHARE;
     }
 
+    /**
+     * @notice Get the fee for flash loans
+     * @return flashloanFee
+     */
     function getFlashLoanFee() external view returns (uint256 flashloanFee) {
         return _flashLoanFee;
     }
 
-    function getLBPairImplementation() external view returns (address LBPairImplementation) {
+    /**
+     * @notice Get the address of the LBPair implementation
+     * @return lbPairImplementation
+     */
+    function getLBPairImplementation() external view returns (address lbPairImplementation) {
         return _lbPairImplementation;
     }
 
-    /// @notice View function to return the number of LBPairs created
-    /// @return The number of LBPair
-    function getNumberOfLBPairs() external view override returns (uint256) {
+    /**
+     * @notice View function to return the number of LBPairs created
+     * @return lbPairNumber
+     */
+    function getNumberOfLBPairs() external view override returns (uint256 lbPairNumber) {
         return _allLBPairs.length;
     }
 
-    function getLBPairAtIndex(uint256 index) external view returns (ILBPair pair) {
+    /**
+     * @notice View function to return the LBPair created at index `index`
+     * @param index The index
+     * @return lbPair The address of the LBPair at index `index`
+     */
+    function getLBPairAtIndex(uint256 index) external view returns (ILBPair lbPair) {
```

Comment:
Inconsistency between `get[...]AtIndex(uint256 index)` and `get[...](uint256 index)`